README.org

```diff
@@ -235,6 +235,9 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
    - [[https://github.com/lukhas/buffer-move][buffer-move]] - Move or swap buffer to other window.
    - [[https://github.com/aki2o/emacs-pophint][emacs-pophint]] - Provide navigation like the Vimperator/KeySnail Hint Mode of Firefox.
    - [[https://github.com/noctuid/vertigo.el][vertigo.el]] - Jump lines using the home row.
+   - [[https://github.com/nixin72/block-nav.el][block-nav]] - Navigate by indentation block levels.
+   - [[https://gitlab.com/ideasman42/emacs-spatial-navigate][emcs-spatial-navigate]] - Navigate by indentation and whitespace blocks.
```

Comment:
emcs-spatial-navigate? typo @clemera ...

Reply:
Thanks, fixed it!

Reply:
👍